docs/cron-workflows.md

```diff
@@ -43,7 +43,8 @@ You can use `CronWorkflow.spec.workflowMetadata` to add `labels` and `annotation
 
 | Option Name                  | Default Value          | Description |
 |:----------------------------:|:----------------------:|-------------|
-| `schedule`                   | None, must be provided | [Cron schedule](#cron-schedule-syntax) to run `Workflows`. Example: `5 4 * * *` |
+| `schedule`                   | None | [Cron schedule](#cron-schedule-syntax) to run `Workflows`. Example: `5 4 * * *`. Either `schedule` or `schedules` must be provided. |
```

Comment:
```suggestion
| `schedule`                   | None | [Cron schedule](#cron-schedule-syntax) to run `Workflows`. Example: `5 4 * * *`. Deprecated, see `schedules`. |
```

We can probably modify this to just say that `schedule` is deprecated. Similar to #13358.

@Joibel does that make sense to you for this feature?

Reply:
Yes, I agree.
It is a trivial change to upgrade and totally automatable. We can then remove the old implementation sooner.